app/controllers/idv/capture_doc_controller.rb

```diff
@@ -34,8 +34,11 @@ def ensure_user_id_in_session
                 document_capture_session_uuid.blank?
 
       result = Idv::DocumentCaptureSessionForm.new(document_capture_session_uuid).submit
+      to_log = result.to_h
+      # Log value used to determine session type ("hybrid flow" or not)
+      to_log[:doc_capture_user_id] = session[:doc_capture_user_id].present?
```

Comment:
Can this be `:doc_capture_user_id_present`? I would expect it to be an actual ID as-is.

Reply:
On a bit of a whim I added a question mark because it seemed Rubyish

Reply:
TIL that if a field has a `?` in it, you need to enclose the field name in backticks to query for it in cloudwatch :-)